installer.py

```
@@ -0,0 +1,30 @@
+    toml_content = f"""
+[server]
+rendezvous_server = "hbbs.cislink.nl:21115"
+relay_server = "hbbr.cislink.nl:21116"
+key = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAICa+NPuA55i85fZSxOljTdjHlQJ04UeZTru7KtFBx6c2 root@n8n-cislink-u35624"
+"""
+    with open(cfg_path, "w") as f:
+        f.write(toml_content.strip())
+
+def run_rustdesk():
+    exe = os.path.join(os.getcwd(), "rustdesk", "rustdesk.exe")
+    subprocess.Popen(exe, shell=True)
```

Ruff (0.15.15)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)